corehq/form_processor/backends/sql/dbaccessors.py

```diff
@@ -221,6 +223,33 @@ def get_reverse_indices(case_id):
         return list(CommCareCaseIndexSQL.objects.raw('SELECT * FROM get_case_indices_reverse(%s)', [case_id]))
 
     @staticmethod
+    def get_all_reverse_indices_info(domain, case_ids):
+        query = CommCareCaseIndexSQL.objects.filter(
+            # domain=domain,  # TODO: Why isn't domain being saved on CommCareCaseIndexSQL objects?
```

Comment:
no specific reason, you could join on case or else add it to CommCareCaseIndexSQL or trust that all case_ids are already in the correct domain.